resolver/from_json.py

```diff
@@ -0,0 +1,23 @@
+import json
+
+from sceptre.exceptions import SceptreException
+from sceptre.resolvers import Resolver
+
+
+class FromJsonResolver(Resolver):
+    def resolve(self):
+        """
+        resolve is the method called by Sceptre. It should carry out the work
+        intended by this resolver. It should return a string to become the
+        final value.
+        """
+        typ = type(self.argument)
+        try:
+            if typ is str:
+                return json.loads(self.argument)
+            else:
+                raise SceptreException(
+                    f"!from_json expects a string argument, got: {typ}"
+                )
```

Comment:
This won't work for nested resolvers when 4.1 comes out. For this to work with nested resolvers, it'll need to take a list with exactly one item.

In other words, you should have the logic be:
1. If the argument is a string, directly load it
2. If the argument is a list, load the first item in it

Also, the Pythonic way to check types is to use `isinstance`, not evaluate `type()` results. It's more polymorphic in nature.

Reply:
I was wondering about that. I can make those changes. I went with the type as a stack variable anticipating testing against it to support other input types. But I think you're right that it breaks polymorphism. I'll switch to `isintance`.